types/are.na/index.d.ts

```diff
@@ -0,0 +1,621 @@
+     * @param params - Additional parameters
+     */
+    channel(
+        indentifier: string | number,
+        params?: Params,
+    ): {
+        /**
+         * Get the channel as an Object. Gets a list of public channels if slug/id not specified. Supports pagination.
+         */
+        get(params?: PaginationParams): Promise<Arena.Channel>;
+        /**
+         * Limited view of the channel
+         */
+        thumb(params?: Params): Promise<Arena.Channel>;
+        /**
+         * Get all of the connections of the channel (channels where this channel is connected). Supports pagination.
+         */
+        connections(params?: PaginationParams): Promise<any>;
+        /**
+         * Get all of the channels connected to blocks in the channel. Supports pagination.
+         */
+        channels(params?: PaginationParams): Promise<Arena.Channel[]>;
+        /**
+         * Get the channel's contents only, as an array. Supports pagination.
+         */
+        contents(params?: PaginationParams): Promise<Arena.Block[]>;
+        /**
+         * Get the channel's collaborators. Supports pagination.
+         */
+        collaborators(params?: PaginationParams): Promise<Arena.User[]>;
+        /**
+         * Creates a new channel. Called as or channel().create(title, status). Title is required, status is optional.
+         */
+        create(title: string, status: Arena.ChannelStatus): Promise<void>;
+        /**
+         * Creates a new channel. Called as channel(title).create(status). Title is required, status is optional.
+         */
+        create(status: Arena.ChannelStatus): Promise<void>;
+        /**
+         * Delete the channel. Can be called as channel(slug).delete() or channel().delete(slug).
+         */
+        delete(slug: string): Promise<void>;
+        /**
+         * Update the channel's attributes. params should be an object and can include title and/or status.
+         * Currently it appears that the are.na API requires both values.
+         * If title is not set, an error will occur.
+         * If status is not set, it will default to "public".
+         */
+        update(params: { title?: string; status?: Arena.ChannelStatus }): Promise<Arena.Channel>;
+        /**
+         * Add collaborators to a channel. Pass userIds as an Array or multiple arguments.
+         */
+        addCollaborators(userIDs: string[]): Promise<Arena.User[]>;
+        /**
+         * Remove collaborators from a channel. Accepts userIds in the same format as addCollaborators.
+         */
+        deleteCollaborators(userIDs: string[]): Promise<Arena.User[]>;
+        /**
+         * Create a block and add it to the channel. Specify textual content or a source link.
+         */
+        createBlock(content: any): Promise<Arena.Block>;
+        /**
+         * Remove a block from the channel
+         */
+        deleteBlock(blockID: number): Promise<void>;
```

Comment:
`blockID` is string
https://github.com/ivangreene/arena-js/blob/master/index.js#L132